src/polyfem/State.cpp

```diff
@@ -1452,6 +1452,7 @@ namespace polyfem
 		// if (args["boundary_conditions"]["rhs"].is_string())
 		// 	rhs_path = resolve_input_path(args["boundary_conditions"]["rhs"]);
 
+		// only has effect for ElasticProblem 
```

Comment:
I think my reasoning was that the p_params object only contains fields for "formulation", "path", and "bbox_center". Then, p_params is passed to problem->set_parameters. However, from looking through the set_parameters definitions for the different problem classes, it seems like the only one that would do anything is the ElasticProblem. The others (as far as I can tell) are a sequences of if statements testing to see if the params object contains certain fields and doing something if certain fields exist. However, the only Problem subclass I found that had an if statement for the parameters defined in p_params was ElasticProblem.

Upon closer look, the TestProblem class may also do something. I'm not exactly sure what merge_patch does. Other than that, though, it still seems to me like this would have no effect in most cases. What am I missing?